core/src/main/scala/org/bitcoins/core/wallet/signer/Signer.scala

```diff
@@ -119,271 +130,248 @@ sealed abstract class Signer {
         BaseTxSigComponent(unsignedTx, index, spendingInfo.output, flags)
     }
   }
+
+  protected def updateScriptSigInSigComponent(
+      unsignedTx: Transaction,
+      inputIndex: Int,
+      output: TransactionOutput,
+      signatureF: Future[ScriptSignature])(
```

Comment:
please name `scriptSignatureF` as I would expect `signatureF` to be a `Future[ECDigitalSignature]`